### PR TITLE
feat: 🎸 SQFormGuidedWorkflow additional section

### PR DESCRIPTION
**Pull request by 20BBrown14**
Added the ability for the SQFormGuidedWorkflow component to have an
additional section above the agent script section.

✅ Closes: #586

Loom: https://www.loom.com/share/89ea19daa1ec4588bbc24ca2db0505c2

### CODE REVIEW

**Comment by laurelbean** [comment]
I wonder if we need to add a test case for this? On one hand, it's a new feature so it should probably add a test. On the other hand, I don't see how this feature would really change how a user uses Guided Workflow?

**Comment by 20BBrown14** [comment]
> I wonder if we need to add a test case for this? On one hand, it's a new feature so it should probably add a test. On the other hand, I don't see how this feature would really change how a user uses Guided Workflow. Might be too close to testing implementation detail

Ya, not sure that'd be useful. we're not testing any "displays x if y" scenarios with the contents of the guided workflow. Instead we test the workflow details (Click x opens section y, etc).

File: src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js [hunk 1]
```diff
@@ -63,6 +64,7 @@ function SQFormGuidedWorkflow({
     useManageTaskModules(initialCompletedTasks, taskModulesContext);
 
   const transformedTaskModules = taskModules.map((taskModule, index) => {
+    console.log('taskModule', taskModule.additionalInformationSectionProps);
```
Comment (SeanGroff) [inline src/components/SQFormGuidedWorkflow/SQFormGuidedWorkflow.js:67]: Stray console.log

File: src/components/SQFormGuidedWorkflow/AdditionalInformationSection.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+function AdditionalInformationSection({
+  actionButton,
+  title,
+  infoText,
+  warningText,
+  errorText,
+  successText,
+  isFailedState,
+  Elements,
```
[the remark this answers is not in the document]
Comment (SeanGroff) [reply src/components/SQFormGuidedWorkflow/AdditionalInformationSection.js:14]: Nevermind. After looking at the consumption story I think `Elements` fits with the existing properties

**Review by SeanGroff — APPROVED** [review]
Approved pending the console.log removal

**Pull request merged**